package.json

```diff
@@ -17,7 +17,15 @@
     "prestart": "node bin/check-nvmrc.js",
     "build": "node tasks/build.js",
     "start": "node tasks/serve.js",
-    "lint": "gulp scss:lint"
+    "test": "npm run lint",
+    "lint": "standard && npm run lint:scss",
+    "lint:scss": "gulp scss:lint"
+  },
+  "standard": {
+    "ignore": [
+      "/lib/",
```

Comment:
Why are we ignoring lib?

Reply:
wasn't sure we keep the same js rules for frontend and backend. I can fix all the errors and remove it from the ignore array.

Reply:
updated